### ISSUE TITLE
New logo is broken on Windows

### ISSUE THREAD

**Issue by Stanzilla** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#3069]
That looks a bit broken on Windows i.imgur.com/X1hEt3p.png

**Comment by JoshuaGarrison27** [comment]
Is it just windows 10? 

Windows 7: 
![image](https://user-images.githubusercontent.com/1706923/39993962-c23dc2fc-5745-11e8-8443-8a5fedcdf08b.png)

**Comment by Stanzilla** [comment]
@JoshuaGarrison27 you are on a version that still has the old icon.

**Comment by JoshuaGarrison27** [comment]
@Stanzilla I just uninstalled, deleted the `C:\Users\{UserName}\AppData\Local\GPMDP_3` folder and then reinstalled it. I still have the same icon as above. 

Interestingly, enough...Here is the new GPMDP_3 folder. I see another icon in there but not the same as the icon that you show.

![image](https://user-images.githubusercontent.com/1706923/40010020-981c3d74-5771-11e8-96ba-e373b1c187b6.png)

**Comment by jostrander** [comment]
@JoshuaGarrison27 The latest published version still has the logo that you see on Update.exe. The app.ico icon is an artifact either from a previous release or something else unrelated. You won't see the new icon until then next version of the player is released.

**Comment by Stanzilla** [comment]
@JoshuaGarrison27 yeah you are on the normal release version, we're on current `master`

**Comment by JoshuaGarrison27** [comment]
Ah okay. My bad. I thought I would be adding value by providing details that things seemed fine on Windows 7. Im actually just in the way now. :smile: Sorry

**Comment by EpicWolverine** [comment]
I just get 4.6.0 today and my taskbar icon looks just like @Stanzilla's screenshot. It looks correct everywhere else (nice redesign by the way). Windows 10 latest update.

**Comment by EpicWolverine** [comment]
I'm not sure what fixed it (I happened to update my graphics driver and reboot) but the problem seems to have disappeared.

**Comment by Restioson** [comment]
Same for me... 
![image](https://user-images.githubusercontent.com/6688948/41995806-3757bf36-7a53-11e8-8419-78790ee34ce4.png)

**Comment by CorruptComputer** [comment]
Mine only does that when the program is not open.
![image](https://user-images.githubusercontent.com/5573038/42002089-ab3a7bf0-7a2b-11e8-9862-db04a7b24a08.png)
![image](https://user-images.githubusercontent.com/5573038/42002106-ba0cea0a-7a2b-11e8-9f79-a0565920ba0d.png)

**Comment by Stanzilla** [comment]
I compiled master and it is not fixed there :(

**Comment by MarshallOfSound** [comment]
@Stanzilla Windows has an icon cache, you need to either

* Completely remove GPMDP from your machine (all executables and all shortscuts) then restart then build from master and check it out
* Remove your windows icon cache, restart and then build from master
